app/assets/javascripts/dashboard/gallery_dataset_view.js

```diff
@@ -75,12 +75,12 @@ class GalleryDatasetView extends React.PureComponent<Props, State> {
       .entries()
       .map(([organization, datasets]) =>
         // Sort each group of datasets
-        [organization, datasets.sort(Utils.localeCompareBy(([]: DatasetType[]), "created", false))],
+        [organization, datasets.sort(Utils.compareBy(([]: DatasetType[]), "created", false))],
       )
       .value()
       .sort(
         // Sort groups by creation date of first dataset
-        Utils.localeCompareBy(
+        Utils.compareBy(
           ([]: DatasetType[]),
```

Comment:
Can you change this line to `([]: Array<[string, APIDatasetType]>)`? Then, the old error should have been caught in my opinion.

Reply:
I wonder why flow didn't complain here in the first place :thinking:

Reply:
Yes I'm wondering that as well, I'll investigate :)